gtk/src/auto/notebook.rs

```diff
@@ -527,6 +527,9 @@ pub trait NotebookExt: 'static {
     #[doc(alias = "gtk_notebook_set_action_widget")]
     fn set_action_widget(&self, widget: &impl IsA<Widget>, pack_type: PackType);
 
+    #[doc(alias = "gtk_notebook_set_current_page")]
+    fn set_current_page(&self, page_num: i32);
```

Comment:
This is manually implemented: https://gtk-rs.org/gtk3-rs/git/docs/src/gtk/notebook.rs.html#415